rdr_service/gcloud_functions/genomic_ingest_data_files_function/main.py

```diff
@@ -44,16 +44,43 @@ def run(self):
 
         _logger.info(f"Event payload: {self.event}")
 
-        allowed_files = ['hard-filtered.gvcf.gz', 'hard-filtered.gvcf.gz.md5sum']
-
-        file_path = self.event.attributes.objectId.replace(" ", "")
-        ext = file_path.split('.', 1)[-1]
-
-        # prod buckets for gvcf files / Only attaching gvcf notifications for now
+        # Files list
+        # red/green idat
+        # red/green idat md5
+        # raw/regular vcf
+        # raw/regular vcf
+        # raw/regular vcf md5
+        # hard-filtered vcf
+        # hard-filtered vcf index
+        # hard-filtered vcf md5
+        # cram
+        # cram md5
+        # cram index
+
+        # prod buckets for gvcf files
         # prod-genomics-data-baylor/Wgs_sample_raw_data/SS_VCF_research/
         # prod-genomics-data-baylor/Wgs_sample_raw_data/SS_VCF_research/
         # prod-genomics-data-northwest/wgs_sample_raw_data/ss_vcf_research/
 
+        allowed_files = [
+            "idat",  # red/green idat
+            "idat.md5sum",  # red/green idat md5
+            "vcf.gz",  # raw/regular vcf
+            "vcf.gz.tbi",  # raw/regular vcf
+            "vcf.gz.md5sum",  # raw/regular vcf md5
+            "hard-filtered.vcf.gz",   # hard-filtered vcf
+            "hard-filtered.vcf.gz.tbi",   # hard-filtered vcf index
+            "hard-filtered.vcf.gz.md5sum",   # hard-filtered vcf md5
+            "cram",   # cram
+            "cram.md5sum",   # cram md5
+            "cram.crai",    # cram index
+            'hard-filtered.gvcf.gz',  # gvcf
+            'hard-filtered.gvcf.gz.md5sum'  # gvcf md5
+        ]
+
+        file_path = self.event.attributes.bucketId + "/" + self.event.attributes.objectId.replace(" ", "")
```

Comment:
Do we need to edit the file path here, since we are still splitting the same way?

Reply:
I needed to add the bucket here so that the full path is included in the DB. I found out late yesterday that `self.event.attributes.objectId` doesn't have the bucket name. Because of this, the full file paths were not being written correctly to the `genomic_gc_validation_metrics` table.

Reply:
Do we need to backfill the table? Was that causing the issue with the files ingestion on not being able to find the metrics record?

Reply:
Sorry didn't scroll down I see the full file path was being sent in the payload.

Reply:
The new table will need to be back-filled. This is something I'll work on while waiting for this to get on production. The missing metrics is a separate issue that is either because the GC uploads the data file before giving us the AW2 manifest, or we haven't ingested the manifest yet. Having the files stored in a table should help mitigate the missing metrics issue, though I don't think it will go away entirely. Either way, the reconciliation will continue to attempt to reconcile them until they are found.